catalog/MDCCatalog/AppDelegate.swift

```diff
@@ -17,7 +17,21 @@ limitations under the License.
 import UIKit
 
 import CatalogByConvention
-import MaterialComponents
+
```

Comment:
Why do we prefer 14 imports to 1?

Reply:
Internally we do not have a global target that includes everything MDC.

Reply:
Why is this an issue now?

Reply:
This is to get our catalog working on internal build.